cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -0,0 +1,115 @@
+
+from charms.reactive import when, when_not, set_state
+from charms.docker import DockerOpts
+from charmhelpers.core import hookenv
+from charmhelpers.core import host
+from charmhelpers.fetch import apt_install
+from charms.templating.jinja2 import render
+
+import os
+import subprocess
+
+
+def _reconfigure_docker_for_sdn():
+    ''' By default docker uses the docker0 bridge for container networking.
+    This method removes the default docker bridge, and reconfigures the
+    DOCKER_OPTS to use the flannel networking bridge '''
+
+    hookenv.status_set('maintenance', 'Reconfiguring docker network bridge')
+    host.service_stop('docker')
+    apt_install(['bridge-utils'], fatal=True)
+    # cmd = "ifconfig docker0 down"
+    # ifconfig doesn't always work. use native linux networking commands to
+    # mark the bridge as inactive.
+    cmd = ['ip', 'link', 'set', 'docker0', 'down']
+    subprocess.check_call(cmd)
+
+    cmd = ['brctl', 'delbr', 'docker0']
+    subprocess.check_call(cmd)
+    set_state('docker.restart')
+
+
+@when('docker.available')
+@when_not('kubernetes.worker.bins.installed')
```

Comment:
You made the status message the same with the master, at some point we should make the states similar. 'kubernetes.worker.bins.installed' -> 'kubernetes_worker_components.installed' ... I don't really care the name just so that the two charms use similar terminology.